packages/neo4j-driver/test/rx/summary.test.js

```diff
@@ -344,15 +344,23 @@ describe('#integration-rx summary', () => {
       runnable = await session.beginTransaction().toPromise()
     }
 
+    let dropUser = true
+
     try {
       await verifySystemUpdates(
         runnable,
-        "CREATE USER foo SET PASSWORD 'bar'",
+        "CREATE USER foo SET PASSWORD 'barizon1'",
         {},
         1
       )
+    } catch (e) {
+      // the user should not be dropped if their creation fails
+      dropUser = !(e instanceof Neo4jError)
```

Comment:
Does `Neo4jError` also cover connectivity errors. Maybe it should be something line 

```js
    try {
      await verifySystemUpdates(
        runnable,
        "CREATE USER foo SET PASSWORD 'barizon1'",
        {},
        1
      )
    } catch (e) {
      // the user should not be dropped if their creation fails
      dropUser = (e instanceof AssertionError) // or whatever error type you expect when the failure does not come from the driver but the tests in `verifySystemUpdates`
      throw e
    } finally {
      if (dropUser) {
        await verifySystemUpdates(runnable, 'DROP USER foo', {}, 1)
      }
    }
```